src/modules/75-cf/components/FlagArchiving/useArchiveFlagDialog.tsx

```diff
@@ -46,36 +107,65 @@ const useArchiveFlagDialog = ({
     }
   }
 
+  const getDialogContent = () => {
+    if (state === STATUS.error)
+      return <PageError message={showError(dependentFlagsError)} onClick={() => refetchDependentFlags()} />
+
+    if (state === STATUS.loading) return <ContainerSpinner flex={{ align: 'center-center' }} />
+    if (state === STATUS.hasDependentFlags) {
+      return (
+        <CannotArchiveWarning
+          flagIdentifier={flagData.identifier}
+          flagName={flagData.name}
+          dependentFlagsResponse={dependentFlagsResponse as Features}
+          queryParams={dependentFlagQueryParams as GetDependentFeaturesQueryParams}
+          pageNumber={pageNumber}
+          setPageNumber={setPageNumber}
+        />
+      )
+    }
+    if (state === STATUS.hasNoDependentFlags) {
```

Comment:
Make sure to use line breaks around conditions like this to break up the wall of code and make it visually scannable.
```suggestion
    if (state === STATUS.loading) return <ContainerSpinner flex={{ align: 'center-center' }} />
    
    if (state === STATUS.hasDependentFlags) {
      return (
        <CannotArchiveWarning
          flagIdentifier={flagData.identifier}
          flagName={flagData.name}
          dependentFlagsResponse={dependentFlagsResponse as Features}
          queryParams={dependentFlagQueryParams as GetDependentFeaturesQueryParams}
          pageNumber={pageNumber}
          setPageNumber={setPageNumber}
        />
      )
    }
    
    if (state === STATUS.hasNoDependentFlags) {
```